src/bp/ui-studio/src/web/translations/en.json

```diff
@@ -47,6 +47,16 @@
     "switchLang": "Change the bot content language. Currently editing: {currentLang}"
   },
   "studio": {
+    "condition": {
+      "label": "Condition",
+      "optionsVariablePlaceholder": {
+        "intent": "intent",
+        "channel": "channel",
+        "language": "language",
+        "topic": "topic"
```

Comment:
Please put single word translation in ui-shared as a top level variable, otherwise we just end up translating the same word a couple of time in a bunch of different places

Reply:
These ones need to be lowercase for the purpose I need them tho, should I make a "lower case" index in shared?